Game.cs

```diff
@@ -1,9 +1,50 @@
 
 class Game
 {
-  public void Run(){
-    Grid grid = new Grid(10, 10);
-    ASCIIGraphics graphics = new ASCIIGraphics();
-    graphics.Draw(grid);
+  public void Run()
+  {
+    TerminalInputOutput input_output = new TerminalInputOutput();
+    LoadSave load_save = new();
```

Comment:
This line might be moved either before `TerminalInputOutput input_output = new TerminalInputOutput();` or after `Args args = input_output.ParseInput();`.
Common rule is to declare variables as locally as possible. That is, you should declare `input_output` just before using it.